src/scicat_online_ingestor.py

```diff
@@ -164,3 +164,6 @@ def main() -> None:
                 # check if we need to commit the individual message
                 if config.kafka.individual_message_commit:
                     _individual_message_commit(offline_ingestors, consumer, logger)
+
+if __name__ == "__main__":
+    main()
```

Comment:
We don't need it cause we expose ``main`` function as ``scicat_ingestor`` command.
You can just type ``scicat_ingestor`` to start the online ingestor.
(it doesn't hurt to have it though)